packages/framework/src/hooks/useRegisterBindings.ts

```diff
@@ -23,6 +24,9 @@ export const useRegisterBindings = <T extends Record<string, unknown>>(
 ): RegisterBindingsResult<T> => {
   const { resolvedWidgetId, rootRef } = useWidgetId(id);
   const [widgetState, setWidgetState] = useWidgetState<T>(resolvedWidgetId);
+  const themeContext = useThemeContext(
+    (values?.styles as Record<string, unknown>)?.names as string | string[],
```

Comment:
You shouldn't need so much type casting...this is a sign the pieces are not fitting together right. It's the whole purpose of typing.